data/time_series.go

```diff
@@ -82,6 +99,109 @@ func (f *Frame) TimeSeriesSchema() (tsSchema TimeSeriesSchema) {
 	return
 }
 
+func valueToType(val float64, ftype FieldType) interface{} {
+	switch ftype {
+	case FieldTypeInt8, FieldTypeNullableInt8:
+		return int8(val)
+	case FieldTypeInt16, FieldTypeNullableInt16:
+		return int16(val)
+	case FieldTypeInt32, FieldTypeNullableInt32:
+		return int32(val)
+	case FieldTypeInt64, FieldTypeNullableInt64:
+		return int64(val)
+	case FieldTypeUint8, FieldTypeNullableUint8:
+		return uint8(val)
+	case FieldTypeUint16, FieldTypeNullableUint16:
+		return uint16(val)
+	case FieldTypeUint32, FieldTypeNullableUint32:
+		return uint32(val)
+	case FieldTypeUint64, FieldTypeNullableUint64:
+		return uint64(val)
+	case FieldTypeFloat32, FieldTypeNullableFloat32:
+		return float32(val)
+	case FieldTypeFloat64, FieldTypeNullableFloat64:
+		return val
+	}
+	// FieldTypeString, FieldTypeNullableString, FieldTypeBool, FieldTypeNullableBool, FieldTypeTime, FieldTypeNullableTime
+	// return an error instead?
+	return val
+}
+
+func (f *Frame) FillMissing(fillMissing FillMissing) error {
```

Comment:
Not sure I fully grok this. Can you add Func doc and maybe some comments in the code to help me :-) Maybe it is more appropriate fill missing Time Series or something like this?